webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -37,7 +37,7 @@ class notificationContainer extends React.Component {
     if (error) {
       const { response: { status } = {} } = error;
```

Comment:
I believe in any case of error we should stop the polling.

Reply:
I would suggest for now we just fix the current workflow and later we can add the notification bar to avoid polling? @tbrisker would you agree? we can open a issue on the tracker!

This is also because we will need this PR to merged to test https://github.com/theforeman/foreman/pull/7338/files

Reply:
I would prefer that we make sure notification polling is stopped before we refresh the page, so there isn't another request sent by the browser while it is reloading. It would be good in any case not to continue polling when there is an error.

Reply:
> I believe in any case of error we should stop the polling

but what if there's a 1 time error such as some network issue,
and on the next call everything will work? I think we should do so only on unauthorized.

> I would prefer that we make sure notification polling is stopped before we refresh the page

I think that when we reload the page, the componenWillUnmount will get called with stopPolling, see https://github.com/theforeman/foreman/blob/f020ccb708a1ea0881858e7bcb82581e0fe3916d/webpack/assets/javascripts/react_app/components/notifications/index.js#L47

Reply:
> but what if there's a 1 time error such as some network issue,
> and on the next call everything will work? I think we should do so only on unauthorized.
I disagree, we shouldn't silently ignore errors and continue trying. If there is an issue causing the request to fail, this will spam the logs every 10 seconds (which was also reported by @ananace iirc).  
We can do a follow-up to this PR that will show a toast notification about what failed, but tbh that shouldn't be the notification's job but rather part of generic api handling. The notification itself should stop polling when it hits an error.